plugs/message/html/render/blog.js

```diff
@@ -92,3 +91,7 @@ function BlogCard ({ blog, blobUrl, onClick, color }) {
 
   return b
 }
+
+function canRender (msg) {
+  return msg.value && msg.value.content && msg.value.content.type === 'blog' && isBlog(msg)
```

Comment:
oh, this isn't needed - `isBlog` is _tight_ as bro - it's built with json-schema validation, so you don't need to write all that paranoid code :P

Reply:
haha! The reason I added it was because I was debugging something. But it does need to expose the isBlog function as message.html.canRender for deciding whether to show the post in "show x updates".